### PR TITLE
[stable/spinnaker] Allow value interpolation in `additionalProfileConfigMaps`

### LINKED ISSUE #16953: [stable/spinnaker] Allow value interpolation in `additionalProfileConfigMaps`
**Is your feature request related to a problem? Please describe.**
At the moment there is no way to dynamically modify the `additionalProfileConfigMaps` attribute. We could allow templating here similar to `additionalInitScript`.

**Describe the solution you'd like**
It would be good to be able to use variables in the contents of the config map. In that way we could leverage the helm templating engine to dynamically modify the services configuration. 

My particular need is related to `orca-local.yml`. I am enabling sql persistence and unfortunately I can only obtain the database ip just before running helm.

If this gets implemented, I could do something like:
```yaml
      orca-local.yml: |-
        sql:
          enabled: true
          connectionPool:
            jdbcUrl: jdbc:mysql://{{ orca.sql_ip }}:3306/orca
            user: orca_service
            password: {{ orca.service_db_password }}
            connectionTimeout: 5000
            maxLifetime: 30000
            maxPoolSize: 50
```

And then pass the variables using `-set` when running helm.

**Describe alternatives you've considered**
At the moment I have to modify the `values.yml` file to replace the values.

### PR DESCRIPTION
**Pull request by afcastano**
#### Is this a new chart
No
> NOTE: We're experiencing a high volume of PRs to this repo and reviews will be delayed. Please host your own chart repository and submit your repository to the Helm Hub instead of this repo to make them discoverable to the community. [Here](https://github.com/helm/hub/blob/master/Repositories.md) is how to submit new chart repositories to the Helm Hub.

#### What this PR does / why we need it:
This PR allows value interpolation in the contents of `additionalProfileConfigMaps`. Sometimes values such as ips or secrets are only known at the moment we install the chart, so it would be good to pass those values as `-set` attributes when doing `helm install`.

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #16953

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Comment by afcastano** [comment]
Hello @paulczar @lwander , 
Any chance this gets reviewed soon?

Thanks,
=)

**Comment by Sluggerman** [comment]
Great feature, currently stuck with some ugly workarounds.

**Review by lwander — APPROVED** [review]
LGTM, although I'm probably no longer the best reviewer for these sorts of changes -- have been on a different team since early 2019.

File: stable/spinnaker/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Open source, multi-cloud continuous delivery platform for releasing software changes with high velocity and confidence.
 name: spinnaker
-version: 1.16.1
+version: 1.19.0
```
Comment (costimuraru) [inline stable/spinnaker/Chart.yaml:4]: Why this big bump? @afcastano @paulczar 
Should we keep this more or less in sync with the spinnaker version (eg. 1.16.1?)

**Pull request merged**